Example/Alister-Example-Tests/Alister/ANPrototypingUIKit/ANBaseTableViewHeaderFooterViewTests.m

```diff
@@ -0,0 +1,71 @@
+//
+//  ANBaseTableViewHeaderFooterViewTests.m
+//  Alister-Example
+//
+//  Created by Maxim Eremenko on 8/22/16.
+//  Copyright © 2016 Oksana Kovalchuk. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import <Expecta/Expecta.h>
+#import "ANBaseTableViewHeaderFooterView.h"
+
+@interface ANBaseTableViewHeaderFooterViewTests : XCTestCase
+
+@property (nonatomic, strong) ANBaseTableViewHeaderFooterView* view;
+
+@end
+
+@implementation ANBaseTableViewHeaderFooterViewTests
+
+- (void)setUp
+{
+    [super setUp];
+    self.view = [ANBaseTableViewHeaderFooterView new];
+}
+
+- (void)tearDown
+{
+    self.view = nil;
+    [super tearDown];
+}
+
+
+#pragma mark - Tests
+
+- (void)test_updateWithModel_positive_textLabelTextEqualModel
+{
+    // given
+    NSString* model = @"model";
+    // when
+    [self.view updateWithModel:model];
+    // then
+    expect(self.view.textLabel.text).to.equal(model);
+}
+
+- (void)test_updateWithNullModel_positive_textLabelTextNotEqualModel
```

Comment:
change test type from positive to negative (because we send not valid or bounds values)